### DISCUSSION TITLE
[IntelliSense] `{ expectedtailwindcss(css-lcurlyexpected)` error

### DISCUSSION THREAD

**Discussion by Sof0506** [tailwindlabs/tailwindcss#9856] [Help]
Type: <b>Bug</b>

Hello team,
I'm facing  this error after the last vscode update I'm working with Nextjs framework every time I initialize a new js project it shows multiple unexplained errors, but when I initilize the project to be Typescript the errors disapear in the tsx files but not in the js files such as tailwind config js , it shows errors such as { expectedtailwindcss(css-lcurlyexpected) and at-rule or selector expectedcss(css-ruleorselectorexpected), although everything is done exaclty as I used to do it before the update so what am I missing ? IThe team in Github told me that the issue is relative to the tailwind extension 

Extension version: 0.9.1
VS Code version: Code 1.73.1 (6261075646f055b99068d3688932416f2346dd3b, 2022-11-09T03:54:53.913Z)
OS version: Linux x64 5.4.0-91-generic
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i3-4005U CPU @ 1.70GHz (4 x 1572)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|0, 0, 0|
|Memory (System)|3.77GB (1.29GB free)|
|Process Argv|. --crash-reporter-id ac13e689-88d6-4fe1-8405-a73537e1ca4f|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|cinnamon|
|XDG_CURRENT_DESKTOP|X-Cinnamon|
|XDG_SESSION_DESKTOP|cinnamon|
|XDG_SESSION_TYPE|x11|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
vscccc:30610679
pyindex848:30577860
nodejswelcome1cf:30587006
fc301958:30595537
2e4cg342:30602488
gswce1:30612156
iaj6b796:30608729
dbltrim-noruby:30604474
89544117:30607850

```

</details>

**Comment by bradlc** [comment]
Hey @Sof0506. I've converted this to a discussion until we can identify an issue with the extension (if there is one). Can you provide more details about the error? What file are you seeing it in? A screenshot would be great.

**Reply by wookelvin** [reply]
I have the same issue when I use the recommended way to apply responsive breakpoints in my css: 
```
@media screen(lg){ 
 ... 
 }
```
I currently use tailwindcss@3.3.5",
